pebblo/topic_classifier/README.md

```diff
@@ -1,7 +1,31 @@
 # Topic Classifier
 
-This is Topic Classifier. 
-Currently, we are supporting following Topics:
+The Pebblo Topic Classifier now offers a streamlined set of document categories: Harmful, Financial, Corporate Documents, and Medical. This update aims to simplify classification, improve precision, and enhance performance by focusing on these key areas.
+
+## Current Supported Topics
+The classifier now supports the following four topics:
+
+1. Harmful
+1. Financial
+1. Corporate Documents
+1. Medical
+
+## How to use
+
+```
+from pebblo.topic_classifier.topic_classifier import TopicClassifier
```

Comment:
@gr8nishan 
This isn't directly related to your PR, but I wanted to let you know that the TopicClassifier used to work independently, as noted in this code snippet. However, it seems that some recent imports from another package have caused it to stop working properly. 
As a result, this snippet isn't working as it did before.

cc: @srics 
```
Python 3.9.6 (default, Aug  9 2024, 14:24:13) 
[Clang 16.0.0 (clang-1600.0.26.3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.

>>> from pebblo.entity_classifier.entity_classifier import EntityClassifier
/Users/raj/Projects/pebblo/venv/3.9/lib/python3.9/site-packages/urllib3/__init__.py:35: NotOpenSSLWarning: urllib3 v2 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'LibreSSL 2.8.3'. See: https://github.com/urllib3/urllib3/issues/3020
  warnings.warn(
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/raj/Projects/pebblo/pebblo/entity_classifier/entity_classifier.py", line 8, in <module>
    from pebblo.entity_classifier.custom_analyzer.private_key_analyzer import (
  File "/Users/raj/Projects/pebblo/pebblo/entity_classifier/custom_analyzer/private_key_analyzer.py", line 17, in <module>
    logger = get_logger(__name__)
  File "/Users/raj/Projects/pebblo/pebblo/log/__init__.py", line 69, in get_logger
    log_level = logging.getLevelName(g_config.logging.level.upper())
AttributeError: 'NoneType' object has no attribute 'logging'
>
```